substrate/frame/balances/src/lib.rs

```diff
@@ -767,6 +767,21 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Burn the specified liquid free balance from the origin account.
+		///
+		/// If the origin's account ends up below the existential deposit as a result
+		/// of the burn, the account will be reaped.
+		#[pallet::call_index(10)]
+		#[pallet::weight(T::WeightInfo::burn_allow_death())]
+		pub fn burn_allow_death(
+			origin: OriginFor<T>,
+			#[pallet::compact] value: T::Balance,
```

Comment:
Updated per request.

Will need help/suggestion with the remaining TODOs in the PR summary:
* running benchmarks & updating weights (I cannot trigger the job)
* check whether prdoc is ok

Reply:
I know you said in another PR that current review process isn't sustainable (I assume due to human resources), but is there someone else we can ask to take a look at the PR?

Reply:
@ggwpez sorry for the tag but could you please give a timeline when this can be checked out? Can we get someone else to review it?

Reply:
Yea, it is best to ask in chat for review. We have the open `#fellowship-open-channel:parity.io` channel for that. I invited you.  
Otherwise it can get lost or not have any priority...

Reply:
Thanks a lot!